src/m_menu.c

```diff
@@ -4150,40 +4150,40 @@ setup_menu_t gen_settings4[] = {
   {"Camera Curve", S_THERMO, m_scrn, CNTR_X, M_THRM_SPC,
    {"joy_response_curve_camera"}, 0, I_ResetController, str_curve},
 
-  {"Movement Deadzone", S_THERMO, m_scrn, CNTR_X, M_THRM_SPC,
+  {"Movement Deadzone", S_THERMO|S_PCT, m_scrn, CNTR_X, M_THRM_SPC,
    {"joy_deadzone_movement"}, 0, I_ResetController},
 
-  {"Camera Deadzone", S_THERMO, m_scrn, CNTR_X, M_THRM_SPC,
+  {"Camera Deadzone", S_THERMO|S_PCT, m_scrn, CNTR_X, M_THRM_SPC,
    {"joy_deadzone_camera"}, 0, I_ResetController},
 
   MI_END
 };
 
 setup_menu_t gen_settings5[] = {
 
-  {"Smooth Pixel Scaling", S_YESNO, m_null, M_X, M_Y,
+  {"Smooth Pixel Scaling", S_ONOFF, m_null, M_X, M_Y,
    {"smooth_scaling"}, 0, M_ResetScreen},
 
-  {"Sprite Translucency", S_YESNO|S_STRICT, m_null, M_X, M_SPC, {"translucency"}},
+  {"Sprite Translucency", S_ONOFF|S_STRICT, m_null, M_X, M_SPC, {"translucency"}},
 
-  {"Translucency", S_THERMO|S_ACTION|S_PCT, m_null, M_X_THRM8, M_SPC,
+  {"Translucency Filter", S_NUM|S_ACTION|S_PCT, m_null, M_X, M_SPC,
    {"tran_filter_pct"}, 0, M_Trans},
```

Comment:
Okay, controversial change:

![woof0019](https://github.com/fabiangreffrath/woof/assets/56656010/08399821-e718-406b-a211-cfbd1357ab00)


Reasoning:
- Boom docs refer to this as "Translucency Filter". "Translucency" alone is simple but too vague in this context.
- The name "Translucency Filter" is too long to use with a thermo.
- The thermo is too small to use the mouse to return the value to 66%, which is poor usability.
- A bigger thermo would look out of place.
- There's no need to draw attention to this setting compared to everything else on the "Display" tab.
- Based on a quick Doomworld search, the sprite translucency toggle is far more important than fine adjustments to the opacity for walls/sprites.

Reply:
Sounds legit.

Reply:
I like the change.